app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="flex h-full flex-col overflow-y-scroll bg-white shadow-xl  max-w-screen-lg px-4 py-2 mx-auto">
+  <div class="flex-1 overflow-y-auto py-2 px-4 sm:px-6">
+    <div class="flex items-start justify-between">
+      <h2 class="text-lg font-medium text-gray-900" id="slide-over-title">Total Amount for group <%= @group.name %>: <%= @transactions_sum || 0 %> $</h2>
+    </div>
+
+    <div class="mt-8">
+      <div class="flow-root">
+        <ul role="list" class="-my-6 divide-y divide-gray-200">
+          <% @transactions.each do |transaction| %>
+            <li class="flex py-6">
+
+              <div class="ml-4 flex flex-1 flex-col">
+                <div>
+                  <div class="flex justify-between text-base font-medium text-gray-900">
+                    <div>
+                      <h3><%= transaction.name %> </h3>
+                      <span class="text-gray-500 text-sm">created on <%= transaction.created_at.strftime("%B %e, %Y")%> </span>
+                    </div>
+                    <p class="ml-4"><%= transaction.amount %> $</p>
+                  </div>
+                </div>
+              </div>
+
+            </li>
+          <% end %>
+        </ul>
+      </div>
+    </div>
+    <div class="py-6 px-4 sm:px-6">
+      <div class="mt-6">
+        <%= link_to 'Create transaction', new_group_transaction_path(group_id: @group.id), class: "flex items-center justify-center border border-transparent bg-secondColor px-6 py-3 text-base font-medium text-white shadow-sm" %>
```

Comment:
- [ ] As mentioned by the previous reviewer, Kindly make sure this button stick as much as possible to the template design. This should be place at the bottom of the page and looks the one on the image below:

| Template design | Your Design |
| ---------------------- | ------------------ |
| ![Screenshot from 2023-01-14 16-28-57](https://user-images.githubusercontent.com/83042742/212476936-d5001dd4-9653-4929-bd89-ec0e3577de17.png) | ![Screenshot from 2023-01-14 16-13-30](https://user-images.githubusercontent.com/83042742/212476329-2896c35d-3db0-4c95-ba97-c06ef3bd16c4.png) |